nevergrad/optimization/optimizerlib.py

```diff
@@ -1029,12 +1030,17 @@ def __init__(
             assert self.budget is not None, "Either scale or budget must be known in _Rescaled."
             scale = math.sqrt(math.log(self.budget) / self.dimension)
         self.scale = scale
+        self.initialization = (
+            np.zeros(self.dimension)
+            if translate is None
+            else self.parametrization.random_state.normal(0.0, translate, size=self.dimension)
+        )
```

Comment:
That can work (providing the change for 'inverse')
I'm still a bit afraid of what would happen in the case of a bounded domain if we go beyond the boundaries.
May be safer to use self.parametrization.sample() to sample a new point in the domain, but you get less control.

Reply:
"self.parametrization.sample()" looks like a good idea for ensuring that we are inside. But that does not use the float "translate".

Reply:
Yes that's what I meant by "less control" :s